frontend/packages/console-shared/src/components/shortcuts/Shortcut.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import { MouseIcon } from '@patternfly/react-icons';
+
+interface ShortcutProps {
+  children: React.ReactNode;
+  alt?: boolean;
+  click?: boolean;
+  ctrl?: boolean;
+  drag?: boolean;
+  hover?: boolean;
+  keyName?: string;
+  rightClick?: boolean;
+  shift?: boolean;
+}
+
+const Command: React.FC = ({ children }) => (
+  <span>
+    <kbd>{children}</kbd>
+  </span>
+);
+
+const Shortcut: React.FC<ShortcutProps> = ({
+  children,
+  ctrl,
+  shift,
+  alt,
+  keyName,
+  drag,
+  click,
+  rightClick,
+  hover,
+}) => {
+  const isMac = window.navigator.platform.includes('Mac');
+  return (
+    <tr>
+      <td>
+        {!isMac && ctrl && <Command>Ctrl</Command>}
+        {alt && <Command>{isMac ? '⌥ Opt' : 'Alt'}</Command>}
+        {shift && <Command>Shift</Command>}
+        {isMac && ctrl && <Command>⌘ Cmd</Command>}
+        {keyName && <Command>{keyName.toUpperCase()}</Command>}
```

Comment:
Maybe this is the last thing always... I'm thinking of say: `Hover + DELETE` to remove a node / item.

Reply:
`hover` is a special case. but I agree that there could be a `hover` + `key`

Reply:
`DELETE + Hover` feels weird to me.